src/keywords.ts

```diff
@@ -15,6 +15,7 @@ export const keywords = [
     'do',
     'else',
     'false',
+    'finally',
```

Comment:
this will be when the PR lands, but it probably doesn't hurt right now

Reply:
FYI, `moc` 0.11.2 doesn't contain `try`-`finally` yet.

Reply:
Got it. Combining these PRs to reduce the complexity of merging these changes + bumping `moc` across all the tooling.